iset.mm

```diff
@@ -99758,6 +99758,64 @@ A Cauchy sequence (as defined here, which has a rate of convergence
     LXCSVPWEXESVGURVHVPVRWAVPVQCDZVRNDABVIZVQPQVPVSVTWTXAUMVPXJTVRIJXKVQVJQVPVS
     VTWTXAVPVNTVSIJWOAVJQVPVOTVTIJWPBVJQVKVLVM $.
 
+  $( An upper bound for ` { A , B } ` .
+     (Contributed by Jim Kingdon, 20-Dec-2021.) $)
+  maxabsle $p |- ( ( A e. RR /\ B e. RR ) ->
+      A <_ ( ( ( A + B ) + ( abs ` ( A - B ) ) ) / 2 ) ) $=
+    ? $.
```

Comment:
I don't understand the usefulness of that result ?

Reply:
In order to apply theorems like http://us.metamath.org/ileuni/eqsupti.html or http://us.metamath.org/ileuni/supclti.html we're going to use rspcev to get the ∃𝑥 ∈ 𝐴 part with `` x = ( ( ( A + B ) + ( abs ` ( A - B ) ) ) / 2 ) ``.

This theorem (applied twice, once for A and once for B) would turn into ∀𝑦 ∈ 𝐵 ¬ 𝑥𝑅𝑦 (the upper bound part).

maxabslemlub would turn into ∀𝑦 ∈ 𝐴 (𝑦𝑅𝑥 → ∃𝑧 ∈ 𝐵 𝑦𝑅𝑧) (the least upper bound part).